requirements.in

```
@@ -1,5 +1,5 @@
 -c api/runtime/requirements.txt
-aws_cdk.aws_dynamodb
-aws_cdk.pipelines
-aws_cdk.core
-cdk-chalice
+aws-cdk-lib>=2.0.0
+chalice[cdkv2]
+constructs>=10.0.0
```

aws-cdk-lib has `"constructs": "^10.0.0"` in its [peerDependencies](https://github.com/aws/aws-cdk/blob/v2-release/packages/aws-cdk-lib/package.json#L369), so version specifier should not be required.
```suggestion
constructs
```

---

Removed no-op